tests/unit/processor/concatenator/test_concatenator.py

```diff
@@ -143,6 +143,25 @@ def specific_rules_dirs(self):
                 },
                 {"field": {"c": "another one"}, "target_field": "first-second"},
             ),
+            (
+                "ignore missing fields",
+                {
+                    "filter": "field.a",
+                    "concatenator": {
+                        "source_fields": ["field.a", "field.b", "other_field.c"],
+                        "target_field": "target_field",
+                        "separator": "-",
+                        "overwrite_target": False,
+                        "delete_source_fields": False,
+                    },
+                },
+                {"field": {"a": "first"}, "other_field": {"c": "third"}},
+                {
+                    "field": {"a": "first"},
+                    "other_field": {"c": "third"},
+                    "target_field": "first-third",
+                },
+            ),
```

Comment:
I don't think this test makes a lot of sense right now. The field `ignore_missing_fields` is missing here and even if I add it, it doesn't influence the test result. Either `true` or `false` results in a successful test.

Reply:
yes you are right. The concatenator ignores missing source fields by its first implementation. But I'm not sure it is expected.
I implemented the missing fields behavior as in all other missing fields implementations here knowing that this breaks the past behavior